src/pages/tutorials/introduction-to-layer-types.md

```diff
@@ -8,4 +8,89 @@ layout: tutorials.hbs
 
 {{ page.data.description }}
 
-Coming Soon.
\ No newline at end of file
+## Feature Layers
+A [Feature Layer](http://esri.github.io/esri-leaflet/api-reference/layers/feature-layer.html) involves pulling down the feature data for a particular layer in the JSON format from an ArcGIS for Server instance to your web app. This means pulling down the information related to an individual feature such as its vertices as well as the attributes. Esri-Leaflet then takes the JSON data and renders it on your map.
+
+Feature Layers are great because they contain all the attribute information (or a subset of the attributes) which means additional operations such as creating popups are really easy beacuse the app already has all the information is needs. You'll typically want to use feature layers where you only need a handful of features that aren't very complex in terms of their geometry.
+
+The challenge with Feature Layers is that they can potentially require transferring large amounts of data from the server to the client. Because all the vertices and attributes are sent this means you may need to send a really big JSON object containing each and every vertice in a complex polygon. For example you wouldn't want to use a Feature Layer if you were drawing to draw tax parcels for the whole of the USA, however only showing a few parcels would be ok.
```

Comment:
i like what you have, but i would organize the information in the section like this:

1. featureLayer queries a service for the geometry and attributes of each individual feature.  if its an old service that doesn't emit geojson, thats okay.
2. we **only** query the service for features that exist in the current map extent (so it might be okay if the service references a dataset with lots and lots of features
2. developers can trim down the payload by only asking for attributes of interest and by generalizing the geometries that are returned (using `simplifyFactor` in the layer constructor)
3. even after optimizing, depending on the service and browser, there will always be a limit to the amount of features which it makes sense to try and display clientside.